WORKSPACE

```diff
@@ -36,3 +36,13 @@ new_http_archive(
     url = "https://pypi.python.org/packages/source/s/six/six-1.10.0.tar.gz#md5=34eed507548117b2ab523ab14b2f8b55",
 )
 
+git_repository(
+    name = "io_bazel_rules_go",
+    remote = "https://github.com/bazelbuild/rules_go.git",
+    commit = "4374be38e9a75ff5957c3922adb155d32086fe14",
```

Comment:
is this go 1.9 SHA?

Reply:
Yes, that's the head of rules_go. Needed for recent native `proto_library` support.

Reply:
Arguably we should have some of what is in this `WORKSPACE` in https://github.com/envoyproxy/data-plane-api/blob/master/bazel/repositories.bzl#L1 so that we can automate the setup of workspaces such as in the main repo. Don't feel that strongly though, if we don't do this we will need to do it in https://github.com/envoyproxy/envoy/blob/master/bazel/repositories.bzl#L146 instead.

Reply:
Right, I wasn't sure what to do with this git repository. The remaining loads have to be in the WORKSPACE since functions cannot load in bazel.